blocks/single-chain-block/chains/single-chain/default.test.jsx

```diff
@@ -14,7 +14,7 @@ describe('single chain', () => {
       );
 
       expect(wrapper.html()).toMatchInlineSnapshot(
-        '"<div class=\\"sc-htpNat jHQBno\\"><p>one chainz</p></div>"',
+        '"<div class=\\"sc-EHOje fUQYKv\\"><p>one chainz</p></div>"',
```

Comment:
can we maybe update this test to just match for text? updating the styled-component is no fun each time